src/toastfile.rs

```diff
@@ -53,8 +115,9 @@ pub struct Task {
     //   Docker socket, which is usually located at `/var/run/docker.sock`)
     // Must not contain `,` [ref:mount_paths_no_commas]
     // Must be empty if `cache` is enabled [ref:mount_paths_nand_cache]
+    // Can be `host_path:container_path` or a single path if host_path is the same as container_path
```

Comment:
Please add backticks around `host_path` and `container_path`, like this:

```rust
    // Can be `host_path:container_path` or a single path if `host_path` is the same as `container_path`
```

If the line is too long and needs to be wrapped (`toast lint` will let you know), then please indent the wrapped part by two spaces (as above).